webtool/test_correct.py

```
@@ -1671,3 +1671,77 @@ def fake_correct_file(project, b, gjson, context, verify, force, **kw):
+    tdir = tmp_path / "P" / "transkripte"
+    tdir.mkdir(parents=True)
+    for b in ("A", "B"):
+        (tdir / f"{b}.json").write_text(json.dumps({"segments": []}), encoding="utf-8")
+        (tdir / f"{b}.raw.txt").write_text("text", encoding="utf-8")
+        (tdir / f"{b}.segments.txt").write_text("[0:00 - 0:01] text", encoding="utf-8")
+
+    monkeypatch.setattr(correct, "_glossary", lambda *a: "")
+    monkeypatch.setattr(correct, "_context", lambda *a: "")
+    monkeypatch.setattr(correct, "cmd_diarize", lambda *a, **kw: 0)
+    monkeypatch.setattr(correct, "prep_single", lambda *a, **kw: True)
+    monkeypatch.setattr(correct, "correct_ai_single", lambda *a, **kw: None)
```

Ruff (0.16.2)

[warning] Unused lambda argument: `a`

(ARG005)